.github/workflows/CI.yml

```diff
@@ -79,3 +79,5 @@ jobs:
       - name: Cargo fmt and clippy
         run: cargo fmt --check &&  cargo clippy --all-targets --all-features --
           -D warnings
+      - name: Prettier for JS
```

Comment:
Maybe move both clipy and js formatter before test? This will run them even if tests are failing.

Reply:
I have yet another suggestion: let's have a separate workflow for code quality (fmt + clippy) and a separate one for correctness (tests). Only the correctness workflow needs to set up a ScyllaDB cluster.